examples/research_projects/lxmert/demo.ipynb

```diff
@@ -6,7 +6,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "#%pip install-r requirements.txt"
+    "# %pip install-r requirements.txt"
```

Comment:
I did `make style` and then I saw this warning:
```
Skipping .ipynb files as Jupyter dependencies are not installed.
You can fix this by running ``pip install black[jupyter]``
```
So I installed `black[jupyter]` in case we want to format the notebooks(maybe not the case?), then it formated the notebooks...

Reply:
I wonder if we format the notebooks with black, do we?